src/core/src/op/assign.cpp

```diff
@@ -61,6 +61,25 @@ bool op::v3::Assign::visit_attributes(AttributeVisitor& visitor) {
     return true;
 }
 
+bool op::v3::Assign::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const {
```

Comment:
I think, it will be better to move reference call to evavuates map instead of ngraph lib. The current decision will increase ngraph lib size

Reply:
I agree. I'll implement the evaluation in evaluates_map.cpp